sdk_container/src/third_party/coreos-overlay/app-crypt/clevis/files/clevis-dracut.patch

```diff
@@ -0,0 +1,216 @@
+diff --git a/src/luks/systemd/dracut/clevis-pin-sss/meson.build b/src/dracut/clevis-pin-sss/meson.build
+similarity index 100%
+rename from src/luks/systemd/dracut/clevis-pin-sss/meson.build
+rename to src/dracut/clevis-pin-sss/meson.build
+diff --git a/src/luks/systemd/dracut/clevis-pin-sss/module-setup.sh.in b/src/dracut/clevis-pin-sss/module-setup.sh.in
+similarity index 100%
+rename from src/luks/systemd/dracut/clevis-pin-sss/module-setup.sh.in
+rename to src/dracut/clevis-pin-sss/module-setup.sh.in
+diff --git a/src/luks/systemd/dracut/clevis-pin-tang/meson.build b/src/dracut/clevis-pin-tang/meson.build
+similarity index 100%
+rename from src/luks/systemd/dracut/clevis-pin-tang/meson.build
+rename to src/dracut/clevis-pin-tang/meson.build
+diff --git a/src/luks/systemd/dracut/clevis-pin-tang/module-setup.sh.in b/src/dracut/clevis-pin-tang/module-setup.sh.in
+similarity index 100%
+rename from src/luks/systemd/dracut/clevis-pin-tang/module-setup.sh.in
+rename to src/dracut/clevis-pin-tang/module-setup.sh.in
+diff --git a/src/luks/systemd/dracut/clevis-pin-tpm2/meson.build b/src/dracut/clevis-pin-tpm2/meson.build
+similarity index 100%
+rename from src/luks/systemd/dracut/clevis-pin-tpm2/meson.build
+rename to src/dracut/clevis-pin-tpm2/meson.build
+diff --git a/src/luks/systemd/dracut/clevis-pin-tpm2/module-setup.sh.in b/src/dracut/clevis-pin-tpm2/module-setup.sh.in
+similarity index 100%
+rename from src/luks/systemd/dracut/clevis-pin-tpm2/module-setup.sh.in
+rename to src/dracut/clevis-pin-tpm2/module-setup.sh.in
+diff --git a/src/dracut/clevis/clevis-hook.sh.in b/src/dracut/clevis/clevis-hook.sh.in
+new file mode 100755
+index 0000000..91ff2bd
+--- /dev/null
++++ b/src/dracut/clevis/clevis-hook.sh.in
+@@ -0,0 +1,3 @@
++#!/bin/bash
++
++@libexecdir@/clevis-luks-generic-unlocker -l
+diff --git a/src/dracut/clevis/clevis-luks-generic-unlocker b/src/dracut/clevis/clevis-luks-generic-unlocker
+new file mode 100755
+index 0000000..a3b9d62
+--- /dev/null
++++ b/src/dracut/clevis/clevis-luks-generic-unlocker
```

Comment:
Do we make us of `clevis-luks-generic-unlocker` or is this not needed anymore?

Reply:
I think we do not need `clevis-luks-generic-unlocker` anymore.

However, I wonder whether we should remove it here. This `.patch` file, which creates `clevis-luks-generic-unlocker`, is part of the upstream ebuild from GURU, so I thought that it'd be preferable to keep it untouched, to make it easier to update in the future? I guess we could remove `clevis-luks-generic-unlocker` in our `.patch` file which is applied after the upstream `.patch` file, but I wonder why we would do that. `clevis-luks-generic-unlocker` shouldn't be included in our initramfs image anyways because of the logic [here](https://github.com/flatcar/scripts/pull/1560/commits/406ee4c085907fe0fe433c07a0a45c5bb55bf750#diff-b2d6a79ae4b1b7c7a321cf35f7bd69b16a287465501162f1bfb2481604e19ae2R146).

(I did see though that we still create a symlink under `/usr/bin` in the initramfs image, pointing to `clevis-luks-generic-unlocker` on /sysusr/usr. I think that symlink is not needed and I will remove it in the bootengine PR.)

Reply:
Ah, ok, sure, it can stay, thanks for the info.